### DISCUSSION TITLE
NVMe mdadm RAID 1 not recognised

### DISCUSSION THREAD

**Discussion by andreapx** [CloverHackyColor/CloverBootloader#679] [General]
I have installed Debian in RAID 1 (mdadm) on 2 NVMe SSD mounted on 2 PCIe adapters.
The motherboard doesn't support to boot from those disks, so I've tried Clover Bootloader, but it doesn't see the 2 drives (I've placed `NvmExpressDxe.efi` in` EFI\CLOVER\drivers\BIOS` and `EFI\CLOVER\drivers\UEFI`).
What could I do?

**Comment by Morbius01** [comment]
Try to do it 'the other way' : install Linux from Clover.
Clear your SSDs.
Boot from the Clover key
When you are in the main Clover menu, insert your Linux installation USB drive. A new icon should appear: 'Boot [U]EFI Optical from...', or something similar. Select it.
This should start the Linux installation process. Follow the instructions until completion.
Remove your Linux installation USB drive.
Restart your PC, booting from the Clover key.
A new icon should be present on the main Clover menu. Select it. This should start Linux.

**Reply by andreapx** [reply]
> Try to do it 'the other way' : install Linux from Clover.

I haven't thought about that, I'll try, thanks!
